panoptes_aggregation/extractors/nfn_extractor.py

```diff
@@ -30,6 +30,7 @@ def iterable(self, arg):
         return (
             isinstance(arg, collections.abc.Iterable)
             and not isinstance(arg, str)
+            and not isinstance(arg, int)
```

Comment:
I don't think this does anything when written like this as an `int` is not iterable so the first `isinstance` will already filter out `int`s.  In python `str` are iterable so that is why the additional check for `str` is there.

Reply:
Ah, that'd explain why str was explicitly excluded here. Kinda funny that integers aren't iterable but strings are...I mean, I get why, but...numbers...anyway yeah this is extraneous anyway given the explicit `dict` check below, will remove.